graphql.schema.json

```diff
@@ -6017,6 +5981,47 @@
             "isDeprecated": false,
             "deprecationReason": null
           },
+          {
```

Comment:
`ceOpportunities` => `eligibleCeOpportunities` is an unrelated change from https://github.com/greenriver/hmis-warehouse/pull/5013#discussion_r2025130471